lib/linguist/heuristics.yml

```diff
@@ -98,6 +98,16 @@ disambiguations:
   rules:
   - language: FreeBasic
     pattern: '^[ \t]*#(?:define|endif|endmacro|ifn?def|if|include|lang|macro)\s'
+- extensions: ['.bs']
+  rules:
+  - language: Bikeshed
+    pattern: '^(?i:<pre\s+class)\s*=\s*(''|\"|\b)metadata\b\1[^>\r\n]*>'
+  - language: BrighterScript
+    pattern:
+    - (?i:^\s*(?=^sub\s)(?:sub\s*\w+\(.*?\))|(?::\s*sub\(.*?\))$)
+    - (?i:^\s*(end\ssub)$)
+    - (?i:^\s*(?=^function\s)(?:function\s*\w+\(.*?\)\s*as\s*\w*)|(?::\s*function\(.*?\)\s*as\s*\w*)$)
+    - (?i:^\s*(end\sfunction)$)
```

Comment:
Given that Bikeshed is a markup format for language specifications, there's no telling what syntaxes might be documented in a `.bs` file. Hence there's a pretty high risk that BrighterScript's heuristics will errantly match something.

I think we're better off removing BrighterScript's heuristics altogether. BrightScript uses a different file extension, so there's no risk of Linguist confusing the two.

```suggestion
```